tests/test_utils.py

```diff
@@ -29,6 +36,30 @@
 DEPLOYMENT_BRAKET_CIRCUITS_JSON = "deployment_request_dto_with_braket_circuit_test_data.json"
 DEPLOYMENT_QISKIT_CIRCUITS_JSON = "deployment_request_dto_with_qiskit_circuit_test_data.json"
 PROGRAM_JSON = "program_request_dto_test_data.json"
+EXPECTED_ID: int = 3
+JOB_FINISHED_PROGRESS: int = 100
+STANDARD_JOB_NAME: str = "JobName"
+IS_ASYNCHRONOUS: bool = False
+RESULT_TOLERANCE: int = 100
+
+
+def generic_test(
+    app, provider: ProviderName, device: str, input_assembler_language: AssemblerLanguage, is_asynchronous: bool
```

Comment:
is_asynchronous kannst du auf jeden Fall raus aus den parametern nehmen

Reply:
Würde das glaube sogar eigentlich in ein neues file packen.
Ein test_transpile_utils oder so - Aber not sure.

Reply:
naja hätten so die Option alles an Inputs für die Tests eben in der Testklasse mitzugeben - ist für mich da schon an genau dem Platz, wo ich es erwarte, wenn ich einen neuen Test schreiben will - wenn man es in eine neue File macht, würde ich es trotzdem dann als Parameter mitgeben und eben jeden Test für sich konfigurieren zu können

Reply:
Dann zumindest als default version auf true setzen, dann muss man es nicht immer mitgeben.